authors/apps/profiles/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
I think it could be good practice to put the tests in a folder and make it a module so that other contributors working on this app can have their test files inside it.

Reply:
I also noticed inconsistency in error codes at times you used 201 for a successful delete

Reply:
@koechkevin , Thanks for the feedback on error  codes,I will have a look at that

Reply:
@koechkevin ,true,since we are working on the same module with @Ramon3162 ,   we will have our tests in the same folder.

Reply:
@koechkevin ,   have you migrated before attempting to test the feature?